cinema/views.py

```diff
@@ -56,3 +59,30 @@ def get_serializer_class(self):
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderSetPagination(PageNumberPagination):
+    page_size = 1
+    page_size_query_param = "page_size"
+    max_page_size = 3
+
+
+class OrderViewSet(viewsets.ModelViewSet):
+    queryset = Order.objects.all()
+    serializer_class = OrderSerializer
+    pagination_class = OrderSetPagination
+
+    def get_queryset(self):
+        queryset = self.queryset.filter(user=self.request.user)
+        if self.action == "list":
+            queryset = queryset.prefetch_related("tickets__movie_session")
```

Comment:
It would be better to prefetch_related("tickets__movie_session__movie", "tickets__movie_session__cinema_hall") instead of tickets__movie_session. Because there are indepent models which have relations with model MovieSession